lib/upgrade-db.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+var MeshServer = require('strong-mesh-models').meshServer;
+var MeshServiceManager = require('strong-mesh-models').ServiceManager;
+var SQLite3 = require('loopback-connector-sqlite3');
+var async = require('async');
+var debug = require('debug')('upgrade-db');
+var fs = require('fs');
+var path = require('path');
+
+module.exports = maybeUpgradeDb;
+
+function maybeUpgradeDb(baseDir, memoryDbPath, dryRun, callback) {
```

Comment:
I was mistaken earlier, this is not a "maybe do this, if it is needed", it's more of a "do this or die trying"